x/ibc/simulation/genesis_test.go

```diff
@@ -57,6 +57,7 @@ func TestRandomizedGenState(t *testing.T) {
 // TestRandomizedGenState tests the execution of RandomizedGenState
 // without registering the IBC client interfaces and types.
 // We expect the test to panic.
+/*
 func TestRandomizedGenState1(t *testing.T) {
```

Comment:
I don't understand what this test was testing, nor why `simulation.RandomizedGenState(&simState)` is no longer failing.

I suspect it has to do with swapping `exported.ClientState` with `GenesisClientState` in GenesisState, but `GenesisClientState` contains `exported.ClientState`, so don't know what the issue is.

Reply:
added in #6712 . @dauTT do you think it is safe for us to delete this test now or how should it be adjusted to account for the new `GenesisState` behaviour?

Reply:
Realized why this test no longer panics. The codec without registering IBC client interfaces/types can still marshal an empty `GenesisClientState` even though it cannot marshal `exported.ClientState` which is contained in `GenesisClientState`.

I tested this hypothesis with these lines:

```go
testing := struct {    
     exported.ClientState
}{}
simState.Cdc.MustMarshalJSON(clientGenesisState)
simState.Cdc.MustMarshalJSON(testing)
```

Note even though exported.ClientState is nested within `clientGenesisState`, we can marshal without panicing (I suspect this is because it is a couple layers deep and the codec doesn't have to marshal nested exported.ClientState since all the structs are empty anyway)

But it will panic when it tries to marshal the `testing` struct.

This makes sense why it would no longer panic, because the test was only expecting a panic on `clientsGenesis` since it used to be raw `[]exported.ClientState`. All the other empty GenesisStates were still expected to pass even though codec was unregistered since they were wrapped by `Genesis` structs.

Since the new `GenesisClientState` wraps over `exported.ClientState` as well, panicing is no longer expected behavior and I will remove this test case.